src/app/main/main.controller.ts

```diff
@@ -16,16 +17,17 @@ module tourneyTracker {
         public spacingValue: number;
         /* @ngInject */
         constructor(private $scope: IMainScope) {
-            var _this: MainCtrl = this;
 
             this.teamHeight = 30;
             this.spacingValue = 8;
             this.matchUnitSize = this.teamHeight * 2 + this.spacingValue;
             $scope.teams = new Array<Team>();
             $scope.matches = new Array<Match>();
-            $scope.addTeam = (name: string) => { _this.addTeam(name); };
-            $scope.createMatchTree = () => { _this.createMatchTree(); };
-            for(var i = 1; i <= 29; ++i) {
+            $scope.square = (num) => Math.pow(2, num);
```

Comment:
I think this is actually `2^x` not `x^2` so not sure that square actually fits for a name